src/routes/safe/components/Apps/utils.ts

```diff
@@ -274,7 +274,16 @@ export const getAppInfoFromUrl = memoize(
       }
       return res
     } catch (error) {
-      logError(Errors._900, `${res.url}: ${error.message}`, undefined, false)
+      logError(
+        Errors._900,
+        error.message,
+        {
+          extra: {
+            appUrl,
+          },
```

Comment:
`appUrl` serves as a kind of id for safe apps, maybe we can pass that in the _901 above as well?

So in both cases it could be:
```
contexts: {
  safeAppUrl: appUrl
},
extra: {
  // free format
  request, method, msg.data
}
```